SCF-Session-7/sparse/SparseTest.java

```diff
@@ -0,0 +1,102 @@
+package sparse;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class SparseTest 
+{
```

Comment:
Many code blocks of class `SparseMatrix` is not covered under test methods. Code coverage must be done in test class.